src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
It's about separation of concerns. But if you want that much freedom I'd say just declare everything in screens with generic html elements. IMO, components are restrictive - that is the purpose of props and proptype validation.